content/nginx/deployment-guides/amazon-web-services/ec2-instances-for-nginx.md

```diff
@@ -4,181 +4,139 @@ description: Create Amazon Elastic Compute Cloud (EC2) instances for running NGI
+        - By default, AWS adds an outbound rule to allow all traffic to all destinations. You can remove that rule by clicking the **Delete** button.
+        - If you use port 80 for client traffic and health checks from a load balancer, like [AWS Network Load Balancer]({{< relref "high-availability-network-load-balancer.md" >}}), you only need one rule.
+        - If you set up different ports for various tasks, or if you use ports like 443 for HTTPS, adjust them accordingly.
 
-        - If, for example, you have used port 80 both for client traffic and for health checks from a load balancer (for example, [AWS Network Load Balancer]({{< relref "high-availability-network-load-balancer.md" >}})), you need only one rule.
-        - If you have configured separate ports for different purposes, or ports other than 80 (such as 443 for HTTPS), make the appropriate adjustments.
+      In the Destination field, start typing your security group's name or ID. It should be listed under Security Groups. The example here shows **sg-0dd4d3c5284052f99**.
 
-      In the **Destination** field, type the security group's ID, which appears in the **Group ID** column in the upper table (here it's <span style="color:#666666; font-weight:bolder;">sg-3bdbf55d</span>).
+         {{< img src="/img/aws/aws-generic-instance-security-outbound.png" alt="Screenshot of the EC2 Security Group outbound rules page">}}
 
-     <a href="/nginx/images/aws-generic-instance-security-outbound.png"><img src="/nginx/images/aws-generic-instance-security-outbound.png" alt="" width="1024" height="459" class="aligncenter size-full wp-image-56330" style="border:2px solid #666666; padding:2px; margin:2px;" /></a>
+13. To install NGINX software on the instance, first [connect]({{< relref "#connect-to-an-ec2-instance" >}}) to it. Then follow the instructions in the NGINX Plus Admin Guide for [NGINX Open Source]({{< relref "/nginx/admin-guide/installing-nginx/installing-nginx-open-source#prebuilt" >}}) and [NGINX Plus]({{< relref "/nginx/admin-guide/installing-nginx/installing-nginx-plus.md" >}}).
 
-14. To install NGINX software on the instance, [connect](#connect-to-instance) to it, and follow the instructions in the NGINX Plus Admin Guide for [NGINX Open Source]({{< relref "/nginx/admin-guide/installing-nginx/installing-nginx-open-source#prebuilt" >}} and [NGINX Plus]({{< relref "/nginx/admin-guide/installing-nginx/installing-nginx-plus.md" >}}).
+---
 
-<span id="connect-to-instance"></span>
-## Connecting to an EC2 Instance
-To install and configure NGINX Open Source or NGINX Plus on an instance, you need to open a terminal window and connect to the instance over SSH.
+## Connect to an EC2 Instance
+To install and configure NGINX Open Source or NGINX Plus on an instance, open a terminal window and connect to the instance over SSH.
 
 1. Navigate to the **Instances** tab on the EC2 Dashboard if you are not there already.
+2. Click the row for an instance to select it.
```

Comment:
If you change all of the explicit numbers to `1.`, Hugo will automatically enumerate them. It's a useful convention, since it means you can change and re-order sequences without being forced to change the numbers every time.

Reply:
Oh, that's nifty! I will update. 🙂